apps/meteor/ee/server/patches/mergeContacts.ts

```diff
@@ -0,0 +1,56 @@
+import type { ILivechatContact, ILivechatContactChannel, ILivechatContactVisitorAssociation } from '@rocket.chat/core-typings';
+import { License } from '@rocket.chat/license';
+import { LivechatContacts, LivechatRooms } from '@rocket.chat/models';
+import type { ClientSession } from 'mongodb';
+
+import { isSameChannel } from '../../../app/livechat/lib/isSameChannel';
+import { ContactMerger } from '../../../app/livechat/server/lib/contacts/ContactMerger';
+import { mergeContacts } from '../../../app/livechat/server/lib/contacts/mergeContacts';
+import { contactLogger as logger } from '../../app/livechat-enterprise/server/lib/logger';
+
+export const runMergeContacts = async (
```

Comment:
is this function being used somewhere that I couldn't find? or is it planned to be used at some point?

Reply:
nvm I'm dumb.. I forgot how the `patch` function works

Reply:
Yes, it is being used [here](https://github.com/RocketChat/Rocket.Chat/blob/6a1b7c8a7ad074792faa8e799eb498c3b03f1bd6/apps/meteor/ee/server/patches/verifyContactChannel.ts#L50), and the `verifyContactChannel` function is being used in the bridge
Please note that these are patch functions (so that we avoid those ugly callbacks in EE)